src/index.html

```diff
@@ -11,7 +11,214 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" class="form">
+      <fieldset>
+        <legend>Personal information</legend>
+        <div>
+          <label for="surname">Surnmae: </label>
+
+          <input
+            type="text"
+            name="surname"
+            id="surname"
+            autocomplete="off"
+          >
+        </div>
+
+        <div class="form-field">
+          <label for="name">Name: </label>
+
+          <input
+            type="text"
+            name="name"
+            id="name"
+            autocomplete="off"
+          >
+        </div>
+
+        <div class="form-field">
+          <label for="age">How old are You?</label>
+
+          <input
+            type="number"
+            name="age"
+            id="age"
+            autocomplete="off"
+            value="12"
+            min="1"
+            max="100"
+            required
+          >
+        </div>
+
+        <div class="form-field">
+          <label for="date-birth">Full date of birth: </label>
+
+          <input
+            type="date"
+            name="date-birth"
```

Comment:
Name attibute should be camelCase
```suggestion
            name="dateBirth"
```